README.md

```diff
@@ -111,17 +118,44 @@ Write a conditional statement that...
 ### Your solution here:
 8.  Write your javascript solution below
 ```js
-// your answer here
+for (let x = 0; x <= 100; x++) {
```

Comment:
this is question 9's answer, this question checks for a single value, no need for a loop. 
Also it has an issue, FizzBuzz will never execute. you should rewrite it a with the most special case "if (x % 3 == 0 && x % 5 == 0) {
        console.log("FizzBuzz");" in the beginning